Braintree.xcodeproj/project.pbxproj

```diff
@@ -1494,6 +1502,10 @@
 		804698292B27C4D70090878E /* BraintreeShopperInsights */ = {
 			isa = PBXGroup;
 			children = (
+				04AA311F2D07990A0043ACAB /* BTButtonOrder.swift */,
```

Comment:
Are we able to alphabetize the files? Should be a setting in the dorectory in xocde

Reply:
Sure, I can alphabetize